src/results/absorption_chiller.jl

```diff
@@ -27,6 +27,24 @@
 # OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED
 # OF THE POSSIBILITY OF SUCH DAMAGE.
 # *********************************************************************************
+"""
+`AbsorptionChiller` results keys:
+- `size_kw` Power capacity size of the absorption chiller system [kW]
+- `size_ton`
+- `year_one_to_tes_series_ton`
+- `year_one_thermal_consumption_series`
+- `year_one_thermal_consumption_kwh`
+- `year_one_thermal_production_tonhour`
+- `year_one_electric_consumption_series`
+- `year_one_electric_consumption_kwh`
+"""
+
+"""
```

Comment:
@NLaws how do you feel about keeping the function definition, but having a separate list of results keys (above)?

Reply:
I've removed the separate function definitions from the doc strings